app/components/c-productlist-wrapper.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div :class="b()">
+    <div :class="b('breadcrumbs--wrapper')">
+      <span :class="b('breadcrumbs')">Sortiment / Licht</span>
+    </div>
+    <e-heading tag-name="h1" font-weight="bold">{{ productlist.length }} Artikel</e-heading>
```

Comment:
- [ ] I don't think that this should be a `h1`
- [ ] Use a translation key for the content which set's different wording for singular / plural (http://kazupon.github.io/vue-i18n/en/pluralization.html)